launch/tier4_control_launch/launch/control.launch.xml

```diff
@@ -0,0 +1,186 @@
+<launch>
+  <!-- option -->
+  <arg name="vehicle_param_file"/>
+  <arg name="vehicle_id"/>
+  <arg name="enable_obstacle_collision_checker"/>
+  <arg name="lateral_controller_mode"/>
+  <arg name="longitudinal_controller_mode"/>
+
+  <!-- common param path -->
+  <arg name="nearest_search_param_path"/>
+
+  <!-- package param path -->
+  <arg name="trajectory_follower_node_param_path"/>
+  <arg name="lat_controller_param_path"/>
+  <arg name="lon_controller_param_path"/>
+  <arg name="vehicle_cmd_gate_param_path"/>
+  <arg name="lane_departure_checker_param_path"/>
+  <arg name="operation_mode_transition_manager_param_path"/>
+  <arg name="shift_decider_param_path"/>
+  <arg name="obstacle_collision_checker_param_path"/>
+  <arg name="external_cmd_selector_param_path"/>
+
+  <!-- component -->
+  <arg name="use_intra_process" default="false" description="use ROS2 component container intra process communication"/>
+  <arg name="use_multithread" default="false" description="use multithread executor"/>
+
+  <group>
+    <push-ros-namespace namespace="control"/>
+
+    <let name="container_executable" value="component_container" unless="$(var use_multithread)"/>
+    <let name="container_executable" value="component_container_mt" if="$(var use_multithread)"/>
+
+    <node_container pkg="rclcpp_components" exec="$(var container_executable)" name="control_container" namespace="">
+      <!-- trajectory follower -->
+      <composable_node pkg="trajectory_follower_node" plugin="autoware::motion::control::trajectory_follower_node::Controller" name="controller_node_exe" namespace="trajectory_follower">
+        <remap from="~/input/reference_trajectory" to="/planning/scenario_planning/trajectory"/>
+        <remap from="~/input/current_odometry" to="/localization/kinematic_state"/>
+        <remap from="~/input/current_steering" to="/vehicle/status/steering_status"/>
+        <remap from="~/input/current_accel" to="/localization/acceleration"/>
+        <remap from="~/input/current_operation_mode" to="/system/operation_mode/state"/>
+        <remap from="~/output/predicted_trajectory" to="lateral/predicted_trajectory"/>
+        <remap from="~/output/lateral_diagnostic" to="lateral/diagnostic"/>
+        <remap from="~/output/slope_angle" to="longitudinal/slope_angle"/>
+        <remap from="~/output/longitudinal_diagnostic" to="longitudinal/diagnostic"/>
+        <remap from="~/output/control_cmd" to="control_cmd"/>
+        <param from="$(var nearest_search_param_path)"/>
```

Comment:
Have you confirmed `param from` works for `composable_node`? (I assume so, but just in case.)
I remember there was a bug in Galactic, right?

Reply:
Yes, I confirmed [here](https://star4.slack.com/archives/C4P0NSMB5/p1659683096007659?thread_ts=1659679927.505959&cid=C4P0NSMB5). It's internal link, though.

Reply:
Thank you, great!

For others, I'll paste the content below.

```console
❯ cat test.launch.xml
<launch>
  <node_container pkg="rclcpp_components" exec="component_container" name="my_container" namespace="" args="test_args">
    <composable_node pkg="demo_nodes_cpp" plugin="demo_nodes_cpp::ParameterBlackboard" name="board1" namespace="test_namespace">
      <param from="/home/daisuke/test.yaml" />
    </composable_node>
  </node_container>

  <load_composable_node target="my_container">
    <composable_node pkg="demo_nodes_cpp" plugin="demo_nodes_cpp::ParameterBlackboard" name="board2" namespace="test_namespace">
      <param from="/home/daisuke/test.yaml" />
    </composable_node>
  </load_composable_node>
</launch>
```

```console
❯ cat test.yaml
/**:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
```

```console
❯ ros2 param dump /test_namespace/board1
/test_namespace/board1:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false


~/workspace/WebAutoMLEvaluations develop*
❯ ros2 param dump /test_namespace/board2
/test_namespace/board2:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
```